Library/Homebrew/lock_file.rb

```diff
@@ -35,3 +35,15 @@ def create_lockfile
     @lockfile.fcntl(Fcntl::F_SETFD, Fcntl::FD_CLOEXEC)
   end
 end
+
+class FormulaLock < LockFile
+  def initialize(name)
+    super("#{name}.formula")
```

Comment:
I think it would be nice to keep the formulae ones as-is and consider just using the new naming format for those in casks. I don't see the costmetic upsides outweighing the entropy and potential breakage of existing uses.